modules/meta.analysis/R/jagify.R

```diff
@@ -70,7 +77,7 @@ transform.nas <- function(data) {
   data$site_id[is.na(data$site_id)] <- 0
   
   # greenhouse defaults to false (0)
-  data$greenhouse[is.na(data$greenhouse)] <- 1
+  data$greenhouse[is.na(data$greenhouse)] <- 0
```

Comment:
@dlebauer this seemed like a typo, so I changed it from `1` to `0`, could you double-check that?

Reply:
no I think the original logic is correct: if no value is given for greenhouse, assume that the data are not from a greenhouse.

Reply:
so in this case `1` is `not from a greenhouse`? I agree with the original logic, but it was `  data$greenhouse[is.na(data$greenhouse)] <- 1`?